server/service/src/sync/program_schemas/program_enrolment.json

```diff
@@ -8,8 +8,8 @@
           "format": "date-time",
           "type": "string"
         },
-        "enrolmentPatientId": {
```

Comment:
Thats a tricky one and I haven't found a good name yet, what it stores is "the patient id within a certain program". For example:

- HIVCare0001", "HIVCare0002
- HIVTesting0001, HIVTesting0002...

program ID sounds very much like the program name, e.g. HIVCareProgram and don't think its the right name.

Maybe `programEnrolmentId` or just `enrolmentId`

In the UI we can name it as it is best for the user though, i.e. as in the issue.